### PR TITLE
Autoplay has been added to the slick slider on openY alert

### PR DESCRIPTION
**Pull request by siddharth28192**
Original Issue, this PR is going to fix: REPLACE WITH A LINK TO ISSUE ( publicly available )

Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature or this is a bug fix - use 8.x-2.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

## Steps for review

- [ ] Please provide steps for review here.
- [ ] Please provide steps for review here.
- [ ] Please provide steps for review here.
- [ ] Please provide steps for review here.

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

File: modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/build/static/js/main.js [hunk 1]
```diff
@@ -1 +1 @@
... o.header.local&&(n+=o.header.local.length),"undefined"!==typeof o.footer&&"undefined"!==typeof o.footer.global&&(r+=o.footer.global.length),"undefined"!==typeof o.footer&&"undefined"!==typeof o.footer.local&&(r+=o.footer.local.length),Object.assign({},t,{loading:!1,alerts:o,headerPager:n>=2,footerPager:r>=2});case a.b:return Object.assign({},t,{loading:!1,error:e.payload});case s.a:var p={},d={},h={},v={},y={};return t.alerts.hasOwnProperty("header")&&(t.alerts.header.hasOwnProperty("global")&&(d=Object.keys(t.alerts.header.global).map(function(n){return parseInt(t.alerts.header.global[n].id)===parseInt(e.payload)?null:t.alerts.header.global[n]}),"undefined"===typeof p.header?p.header={global:d.filter(Boolean)}:p.header.global=d.filter(Boolean)),t.alerts.header.hasOwnProperty("local")&&(h=Object.keys(t.alerts.header.local).map(function(n){return parseInt(t.alerts.header.local[n].id)===parseInt(e.payload)?null:t.alerts.header.local[n]}),"undefined"===typeof p.header?p.header={local:h.filter(Boolean)}:p.header.local=h.filter(Boolean))),t.alerts.hasOwnProperty("footer")&&(t.alerts.footer.hasOwnProperty("global")&&(v=Object.keys(t.alerts.footer.global).map(function(n){return parseInt(
```
Comment (podarok) [inline modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/build/static/js/main.js]: Sorry, we are not accepting changes to compiled file to distribution. 
Please, add a change to source javascript and ship it with PR.

Thank you

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Review by podarok — CHANGES_REQUESTED** [review]
We should have source files change too.
Please fix.

File: modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/build/static/js/main.js (hunk 1 above)
Comment (siddharth28192) [reply modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/build/static/js/main.js]: I am not able to understand what are you trying to say. I have made changes with main.js file and I was facing carousel issue with alerts, So I made this changes. No idea about source javascript.

Comment (podarok) [reply modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/build/static/js/main.js]: @siddharth28192
See https://github.com/ymcatwincities/openy/tree/8.x-2.x/modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts
There is a yarn/npm workflow for the alerts, built by using ReactJS
We have list of commands in here https://github.com/ymcatwincities/openy/blob/8.x-2.x/modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/package.json#L73
It's a modular application https://github.com/ymcatwincities/openy/tree/8.x-2.x/modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/src
WHich contains components and all needed dependencies that should be updated in order to accomplish what you are trying to do in this PR

I'm asking you not to change compiled file, but to change sources, run `yarn build` , get new main.js and add changed sources to this PR.

Comment (siddharth28192) [reply modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/build/static/js/main.js]: @podarok Thanks for letting me know, how to rebuild main.js file. I will rebuild and create the again PR.

Thankyou!

Comment (podarok) [reply modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/build/static/js/main.js]: Thank YOU!

File: modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/src/AppFooter.js [hunk 2]
```diff
@@ -41,7 +41,9 @@ class App extends Component {
       arrows: false,
       variableWidth: false,
       centerMode: true,
-      centerPadding: '0px'
+      centerPadding: '0px',
+      autoplay:true,
+      autoplaySpeed:5000,
```
Comment (duozersk) [inline modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/src/AppFooter.js]: Could you please add blank symbol after colons?

File: modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/src/AppHeader.js [hunk 3]
```diff
@@ -45,7 +45,9 @@ class App extends Component {
       arrows: false,
       variableWidth: false,
       centerMode: true,
-      centerPadding: '0px'
+      centerPadding: '0px',
+      autoplay:true,
+      autoplaySpeed:5000,
```
Comment (duozersk) [inline modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/src/AppHeader.js]: Could you please add blank symbol after colons?

**Review by duozersk — CHANGES_REQUESTED** [review]

File: modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/src/AppFooter.js (hunk 2 above)
Comment (siddharth28192) [reply modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/src/AppFooter.js]: @duozersk  Thank you for let me know standard of coding.

File: modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/src/AppHeader.js (hunk 3 above)
Comment (siddharth28192) [reply modules/openy_features/openy_node/modules/openy_node_alert/js/openy-alerts/src/AppHeader.js]: @duozersk Thank you for let me know standard of coding.

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by podarok** [comment]
no response and lack of demand
closing for now

**Pull request closed without merging**